### PR TITLE
adding new CLI nuget commands

### LINKED ISSUE #955: New Doc for dotnet nuget push

### LINKED ISSUE #956: New Doc for dotnet nuget delete

### LINKED ISSUE #957: New Doc for dotnet nuget locals

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #955, fixes #956, fixes #957  

/cc @karann-msft

### CODE REVIEW

**Comment by saldana** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/core/tools/dotnet-build.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-build.md) | :white_check_mark:Succeeded | | [Details](#user-content-docs/core/tools/dotnet-build.md)
[docs/core/tools/dotnet.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet.md) | :white_check_mark:Succeeded | | [Details](#user-content-docs/core/tools/dotnet.md)
[docs/current-1.1/tools/dotnet-nuget-delete.md](https://github.com/dotnet/docs/blob/nugetcli/docs/current-1.1/tools/dotnet-nuget-delete.md) | :white_check_mark:Succeeded | | [Details](#user-content-docs/current-1.1/tools/dotnet-nuget-delete.md)
[docs/current-1.1/tools/dotnet-nuget-locals.md](https://github.com/dotnet/docs/blob/nugetcli/docs/current-1.1/tools/dotnet-nuget-locals.md) | :white_check_mark:Succeeded | | [Details](#user-content-docs/current-1.1/tools/dotnet-nuget-locals.md)
[docs/current-1.1/tools/dotnet-nuget-push.md](https://github.com/dotnet/docs/blob/nugetcli/docs/current-1.1/tools/dotnet-nuget-push.md) | :white_check_mark:Succeeded | | [Details](#user-content-docs/current-1.1/tools/dotnet-nuget-push.md)






For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C10%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201611102146160526-1218%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/core/tools/dotnet-build.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-build.md) | :white_check_mark:Succeeded | | [Details](#user-content-docs/core/tools/dotnet-build.md)
[docs/core/tools/dotnet.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet.md) | :white_check_mark:Succeeded | | [Details](#user-content-docs/core/tools/dotnet.md)
[docs/current-1.1/tools/dotnet-nuget-delete.md](https://github.com/dotnet/docs/blob/nugetcli/docs/current-1.1/tools/dotnet-nuget-delete.md) | :white_check_mark:Succeeded | | [Details](#user-content-docs/current-1.1/tools/dotnet-nuget-delete.md)
[docs/current-1.1/tools/dotnet-nuget-locals.md](https://github.com/dotnet/docs/blob/nugetcli/docs/current-1.1/tools/dotnet-nuget-locals.md) | :white_check_mark:Succeeded | | [Details](#user-content-docs/current-1.1/tools/dotnet-nuget-locals.md)
[docs/current-1.1/tools/dotnet-nuget-push.md](https://github.com/dotnet/docs/blob/nugetcli/docs/current-1.1/tools/dotnet-nuget-push.md) | :white_check_mark:Succeeded | | [Details](#user-content-docs/current-1.1/tools/dotnet-nuget-push.md)






For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C10%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201611102242048270-1218%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/core/tools/dotnet-build.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-build.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-build?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-build.md)
[docs/core/tools/dotnet.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet.md)
[docs/current-1.1/tools/dotnet-nuget-delete.md](https://github.com/dotnet/docs/blob/nugetcli/docs/current-1.1/tools/dotnet-nuget-delete.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/current-1.1/tools/dotnet-nuget-delete?branch=pr-en-us-1218) | [Details](#user-content-docs/current-1.1/tools/dotnet-nuget-delete.md)
[docs/current-1.1/tools/dotnet-nuget-locals.md](https://github.com/dotnet/docs/blob/nugetcli/docs/current-1.1/tools/dotnet-nuget-locals.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/current-1.1/tools/dotnet-nuget-locals?branch=pr-en-us-1218) | [Details](#user-content-docs/current-1.1/tools/dotnet-nuget-locals.md)
[docs/current-1.1/tools/dotnet-nuget-push.md](https://github.com/dotnet/docs/blob/nugetcli/docs/current-1.1/tools/dotnet-nuget-push.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/current-1.1/tools/dotnet-nuget-push?branch=pr-en-us-1218) | [Details](#user-content-docs/current-1.1/tools/dotnet-nuget-push.md)






For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C11%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201611111821538078-1218%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/current-1.1/tools/dotnet-nuget-delete.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ms.devlang: dotnet
+ms.assetid: 6ddffde4-c789-4e90-990e-d35f6a6565d4
+---
+
+#dotnet-nuget-delete
+
+[!INCLUDE[preview-warning](../../includes/warning.md)]
+
+## Name 
+dotnet-nuget-delete -- Deletes or unlists a package from the server.
```
Comment (karann-msft) [inline docs/current-1.1/tools/dotnet-nuget-delete.md]: put dotnet-nuget-delete in a code block and remove the extra hyphen (similar to [dotnet-pack](https://docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-pack))

File: docs/current-1.1/tools/dotnet-nuget-delete.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+`-k|--api-key <API_KEY>`
+
+The API key for the server.
+
+`--force-english-output`
+
+Forces command-line output to be in English.
+
+`--verbosity`
+
```
Comment (karann-msft) [inline docs/current-1.1/tools/dotnet-nuget-delete.md]: For your question in the email, --verbosity can have three possible values - normal, quiet, detailed

File: docs/current-1.1/tools/dotnet-nuget-locals.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+ms.devlang: dotnet
+ms.assetid: 8440229e-317e-4dc1-9463-cba5fdb12c3b
+---
+
+#dotnet-nuget-locals
+
+[!INCLUDE[preview-warning](../../includes/warning.md)] 
+
+## Name 
+dotnet-nuget-locals -- Clears or lists local NuGet resources such as http-request cache, temporary cache, or machine-wide global packages folder.
```
Comment (karann-msft) [inline docs/current-1.1/tools/dotnet-nuget-locals.md]: put dotnet-nuget-locals in a code block and remove the extra hyphen (similar to [dotnet-pack](https://docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-pack))

File: docs/current-1.1/tools/dotnet-nuget-push.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+ms.devlang: dotnet
+ms.assetid: f54d9adf-94f8-41cc-bb52-42f7ca3be6ff
+---
+
+#dotnet-nuget-push
+
+[!INCLUDE[preview-warning](../../includes/warning.md)]
+
+## Name 
+dotnet-nuget-push -- Pushes a package to the server and publishes it.
```
Comment (karann-msft) [inline docs/current-1.1/tools/dotnet-nuget-push.md]: put dotnet-nuget-push in a code block and remove the extra hyphen (similar to [dotnet-pack](https://docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-pack))

File: docs/current-1.1/tools/dotnet-nuget-push.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+[!INCLUDE[preview-warning](../../includes/warning.md)]
+
+## Name 
+dotnet-nuget-push -- Pushes a package to the server and publishes it. 
+
+## Synopsis
+
+`dotnet nuget push [<package>] [--help] [--source] [--symbols-source] 
+    [--timeout] [api-key] [--symbol-api-key] [--disable-buffering] [--no-symbols]
```
Comment (karann-msft) [inline docs/current-1.1/tools/dotnet-nuget-push.md]: [api-key] should be [--api-key]

File: docs/current-1.1/tools/dotnet-nuget-push.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+`--symbols-source <SOURCE>`
+
+Specifies the symbol server URL.
+
+`-t|--timeout <TIMEOUT>`
+
+Specifies the timeout for pushing to a server in seconds. Defaults to 300 seconds (5 minutes). Specifying 0 also gives this default.
+
+`-k|api-key <API_KEY>`
```
Comment (karann-msft) [inline docs/current-1.1/tools/dotnet-nuget-push.md]: api-key should be --api-key

File: docs/current-1.1/tools/dotnet-nuget-push.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+## Synopsis
+
+`dotnet nuget push [<package>] [--help] [--source] [--symbols-source] 
+    [--timeout] [api-key] [--symbol-api-key] [--disable-buffering] [--no-symbols] 
+    [--force-english-output] [--config-file] [--verbosity]`
+
+## Description
+
+The `dotnet nuget push` command pushes a package to the server and publishes it. 
+The push command uses server and credential details found in the system's NuGet config file, or chain of config files (see <here> for more information on config files).
```
Comment (karann-msft) [inline docs/current-1.1/tools/dotnet-nuget-push.md]: use this link for <here>
https://docs.nuget.org/ndocs/consume-packages/configuring-nuget-behavior

File: docs/current-1.1/tools/dotnet-nuget-locals.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+`dotnet nuget locals <cache_location> [--clear|--list] [--help] [--force-english-output]`
+
+where `<cache_location>` is one of the following values: `all`, `http-cache`, `packages-cache`, `global-packages`, or `temp`.
+
+## Description
+
+The `dotnet nuget locals` command clears or lists local NuGet resources such as http request cache, temporary cache, or machine-wide global packages folder.
+
+## Arguments
+
```
Comment (karann-msft) [inline docs/current-1.1/tools/dotnet-nuget-locals.md]: consider putting the argument and its description on separate lines, the way you do for options.

**Review by karann-msft — CHANGES_REQUESTED** [review]
A few suggestions for small changes.

File: docs/current-1.1/tools/dotnet-nuget-push.md (hunk 7 above)
Comment (mairaw) [inline docs/current-1.1/tools/dotnet-nuget-push.md]: thanks!

File: docs/current-1.1/tools/dotnet-nuget-push.md (hunk 6 above)
Comment (mairaw) [inline docs/current-1.1/tools/dotnet-nuget-push.md]: fixed

**Comment by mairaw** [comment]
Thanks for your review @karann-msft! I think I've addressed your comments. Please take a look. Also your original document about nuget locals had a link to a troubleshooting guide that doesn't work. Is there an alternative link?

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/core/tools/dotnet-build.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-build.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-build?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-build.md)
[docs/core/tools/dotnet-install-script.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-install-script.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-install-script?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-install-script.md)
[docs/core/tools/dotnet-new.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-new.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-new?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-new.md)
[docs/core/tools/dotnet-pack.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-pack.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-pack?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-pack.md)
[docs/core/tools/dotnet-publish.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-publish.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-publish?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-publish.md)
[docs/core/tools/dotnet-restore.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-restore.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-restore?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-restore.md)
[docs/core/tools/dotnet-run.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-run.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-run?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-run.md)
[docs/core/tools/dotnet-test.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-test.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-test?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-test.md)
[docs/core/tools/dotnet.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet.md)
[docs/current-1.1/tools/dotnet-nuget-delete.md](https://github.com/dotnet/docs/blob/nugetcli/docs/current-1.1/tools/dotnet-nuget-delete.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/current-1.1/tools/dotnet-nuget-delete?branch=pr-en-us-1218) | [Details](#user-content-docs/current-1.1/tools/dotnet-nuget-delete.md)
[docs/current-1.1/tools/dotnet-nuget-locals.md](https://github.com/dotnet/docs/blob/nugetcli/docs/current-1.1/tools/dotnet-nuget-locals.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/current-1.1/tools/dotnet-nuget-locals?branch=pr-en-us-1218) | [Details](#user-content-docs/current-1.1/tools/dotnet-nuget-locals.md)
[docs/current-1.1/tools/dotnet-nuget-push.md](https://github.com/dotnet/docs/blob/nugetcli/docs/current-1.1/tools/dotnet-nuget-push.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/current-1.1/tools/dotnet-nuget-push?branch=pr-en-us-1218) | [Details](#user-content-docs/current-1.1/tools/dotnet-nuget-push.md)













For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C12%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201611120207542917-1218%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Thanks @mairaw 
Here is the current link for the nuget locals troubleshooting guide - https://docs.nuget.org/ndocs/consume-packages/managing-the-nuget-cache

The link in the warning is broken - "see the major release version documentation".

What would the final URL for delete, locals and push look like?

**Comment by mairaw** [comment]
@karann-msft I'll make the change. we're changing the folder name, so I'll give you the final URL later tonight.

**Comment by mairaw** [comment]
Las changes are in @karann-msft. The final URLs should be:
https://docs.microsoft.com/en-us/dotnet/articles/cli-preview3/tools/dotnet-nuget-delete
https://docs.microsoft.com/en-us/dotnet/articles/cli-preview3/tools/dotnet-nuget-locals
https://docs.microsoft.com/en-us/dotnet/articles/cli-preview3/tools/dotnet-nuget-push

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/cli-preview3/tools/dotnet-nuget-delete.md](https://github.com/dotnet/docs/blob/nugetcli/docs/cli-preview3/tools/dotnet-nuget-delete.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/cli-preview3/tools/dotnet-nuget-delete?branch=pr-en-us-1218) | [Details](#user-content-docs/cli-preview3/tools/dotnet-nuget-delete.md)
[docs/cli-preview3/tools/dotnet-nuget-locals.md](https://github.com/dotnet/docs/blob/nugetcli/docs/cli-preview3/tools/dotnet-nuget-locals.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/cli-preview3/tools/dotnet-nuget-locals?branch=pr-en-us-1218) | [Details](#user-content-docs/cli-preview3/tools/dotnet-nuget-locals.md)
[docs/cli-preview3/tools/dotnet-nuget-push.md](https://github.com/dotnet/docs/blob/nugetcli/docs/cli-preview3/tools/dotnet-nuget-push.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/cli-preview3/tools/dotnet-nuget-push?branch=pr-en-us-1218) | [Details](#user-content-docs/cli-preview3/tools/dotnet-nuget-push.md)
[docs/core/tools/dotnet-build.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-build.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-build?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-build.md)
[docs/core/tools/dotnet-install-script.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-install-script.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-install-script?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-install-script.md)
[docs/core/tools/dotnet-new.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-new.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-new?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-new.md)
[docs/core/tools/dotnet-pack.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-pack.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-pack?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-pack.md)
[docs/core/tools/dotnet-publish.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-publish.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-publish?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-publish.md)
[docs/core/tools/dotnet-restore.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-restore.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-restore?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-restore.md)
[docs/core/tools/dotnet-run.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-run.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-run?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-run.md)
[docs/core/tools/dotnet-test.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet-test.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet-test?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet-test.md)
[docs/core/tools/dotnet.md](https://github.com/dotnet/docs/blob/nugetcli/docs/core/tools/dotnet.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/articles/core/tools/dotnet?branch=pr-en-us-1218) | [Details](#user-content-docs/core/tools/dotnet.md)













For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C15%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201611152305015393-1218%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**